pulp_smash/resources/__init__.py

```diff
@@ -0,0 +1,4 @@
+# coding=utf-8
+"""Modules for easy manipulation with pulp resources like users, repositories
+etc."""
```

Comment:
Since it's very likely that big portion of code and it's organization will change I didn't pay much attention to docstrings in here, partly also because I wasn't sure about purpose of each module. I agree about docstrings, one-liner looks better.